changelogs.py

```diff
@@ -0,0 +1,38 @@
+import re
```

Comment:
Is the plan for this to be used by more repos?

Reply:
YES this script will be used by the 'changelogs' GitHub workflow and will be added in other repos as well